pages/osx/lp.md

```diff
@@ -0,0 +1,8 @@
+#lp
+> print files
+- print a file to a named printer (found in Printers & Scanners --> Options and Supplies)
+'lp -d {{device_name}} {{path/to/filename}}
```

Comment:
not sure what you mean??

On Tue, Nov 3, 2015 at 11:08 PM, lord63 <notifications@github.com> wrote:

> In pages/osx/lp.md
> <https://github.com/tldr-pages/tldr/pull/318#discussion_r43841216>:
>
> > @@ -0,0 +1,8 @@
> > +#lp
> > +> print files
> > +- print a file to a named printer (found in Printers & Scanners --> Options and Supplies)
> > +'lp -d {{device_name}} {{path/to/filename}}
>
> wrap command with ` not '.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tldr-pages/tldr/pull/318/files#r43841216>.
>



-- 
Regards,

Michael Quatrani
215.694.3468